internal/moduleconfig/moduleconfig.go

```diff
@@ -333,3 +324,11 @@ func parseImports(filePath string) ([]string, error) {
 	}
 	return imports, nil
 }
+
+func (c ModuleConfig) Schema() string {
```

Comment:
Schema is going to be passed back from plugins over gRPC, so modules themselves shouldn't be configuring the location of the schema file.